docs/ossm/ossm2-migration/README.md

```diff
@@ -8,22 +8,117 @@ Before you begin to migrate your controlplane from OpenShift Service Mesh 2.6 to
 
 - Upgrade your 2.6 OpenShift Service Mesh Operator to the latest release. See warning below.
 - Upgrade your `ServiceMeshControlPlane` version to the latest OpenShift Service Mesh release.
+- Install your 3.0 Openshift Service Mesh Operator. See warning below.
+- Upgrade your Kiali operator to the latest release.
 - Disable the following features on your `ServiceMeshControlPlane`. These fields are unsupported in 3.0 and must be disabled prior to migration.
-    <!-- TODO: create a separate page for each of these bullet points describing how to migrate off the SMCP managed version. -->
-    <!-- TODO: revisit this list when: https://issues.redhat.com/browse/OSSM-8299 is completed. -->
   - Network Policy management: `spec.security.manageNetworkPolicy=false`. If you wish to keep the Network Policies created by the 2.6 `ServiceMeshControlPlane`, you will need to recreate and manage these manually.
-  - Disabled addons:
-    - Prometheus: `spec.addons.prometheus.enabled=false`. See [here](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/metrics-and-service-mesh#ossm-config-openshift-monitoring-only_ossm-metrics-assembly) for instructions on how to configure OpenShift Service Mesh 3.0 with OpenShift Monitoring as a replacement for the prometheus addon.
-    - Kiali: `spec.addons.kiali.enabled=false`. See [here](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/kiali-operator-provided-by-red-hat#ossm-install-kiali-operator_ossm-kiali-assembly) for instructions on how to install and configure Kiali with OpenShift Service Mesh 3.0 as a replacement for the Kiali addon.
-    - Grafana: `spec.addons.grafana.enabled=false`
-    - Tracing: `spec.tracing.type=None`. See [here](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/distributed-tracing-and-service-mesh#ossm-distr-tracing-assembly) for instructions on how to configure OpenShift Service Mesh 3.0 with OpenShift Distributed Tracing as a replacement for the tracing addon.
+  - Disabled addons. The OpenShift Service Mesh 3.0 operator does not manage addons. If these are enabled on your `SerivceMeshControlPlane`, you will now need to manage these yourself by first migrating to an external instance and then disabling the addon.
```

Comment:
I don't know if the term "addon" will be that familiar to users. Could we list all possible addons here?

Reply:
Oops, looks like the list is below - sorry.